src/FakeItEasy/Creation/IFakeCreatorFacade.cs

```diff
@@ -18,8 +18,17 @@ internal interface IFakeCreatorFacade
         T CreateFake<T>(Action<IFakeOptions<T>> optionsBuilder);
 
         /// <summary>
+        /// Creates a fake object of the specified type.
+        /// </summary>
+        /// <param name="typeOfFake">The type of fake to create.</param>
+        /// <param name="optionsBuilder">Action that builds options for the created fake object.</param>
+        /// <returns>The created fake object.</returns>
+        /// <exception cref="FakeItEasy.Core.FakeCreationException">Was unable to generate the fake in the current configuration.</exception>
+        object CreateFake(Type typeOfFake, Action<IFakeOptions> optionsBuilder);
+
+        /// <summary>
         /// Creates a dummy object, this can be a fake object or an object resolved
-        /// from the current IFakeObjectContainer.
+        /// from a dummy factory.

```

Comment:
There seems to be a few places where `IFakeObjectContainer` is still mentioned... we should probably make a sweep to fix it everywhere.